spec/descriptions/tagInfrastructureAnalyze.md

```diff
@@ -0,0 +1,45 @@
+This endpoint group exposes the functions that used by the Instana **Analyze Infrastructure** dashboards.
+Two of the endpoints provide a [list of entity types](#operation/getAvailablePlugins) and [metrics for an entity type](#operation/getAvailableMetrics).
+It is also possible to [search and filter entities](#operation/getEntities) and [aggregate metrics on similar entities](#operation/getEntityGroups).
+
+## Common Parameters
+
+### Mandatory Parameters
+
+**timeFrame** As in the Instana dashboards, you can specify the timeframe for metrics retrieval.
+```
+  windowSize           to
+     (ms)       (unix-timestamp)
+<----------------------|
+```
+The timeFrame might be adjusted to fit the metric granularity so that there is no partial bucket. For example, if the query timeFrame is 08:02 - 09:02 and the metric granularity is 5 minutes, the timeFrame will be adjusted to 08:05 - 09:00. The adjusted timeFrame will be returned in the response payload. If the query does not have any metric with specified granularity, a default granularity will be used for adjustment.
+
+### Optional Parameters
+
+**pagination**
+* *offset* specifies the starting point for the data retrieval
+* *retrievalSize* sets the number of values to return
+* *ingestionTime* this parameter indicates where data retrieval should begin for the next data retrieval request. The results set ingestionTime property from the `cursor` class needs to be used to set this parameter value.
+
+**order**
+* *by* metric name or `label`
+
+**tagFilters** As in the Instana dashboards, you can filter by a tag. Query the [infrastructure tag catalog](#operation/getInfrastructureCatalogMetrics) to retrieve a list of available tags.
+* *name* The name of the tag as returned by the tag catalog
+* *value* The filter value of the tag, possible types are:
+  * "STRING" alphanumerical values.  Valid operators: "EQUALS", "CONTAINS", "NOT_EQUAL", "NOT_CONTAIN", "NOT_EMPTY",  "IS_EMPTY"
+  * "NUMBER" numerical values. Valid operators: "EQUALS", "LESS_THAN" "GREATER_THAN"
+  * "KEY_VALUE_PAIR" 
+* *operator* one of the valid operators for the tag filter type *value*
```

Comment:
**question** ❓ Is **tagfilters** the same as **tagFilterExpression**? The examples show this:

```
"tagFilterExpression": {
    "type": "EXPRESSION",
    "logicalOperator": "AND",
    "elements": []
  },
```

The model shows this:

```
"tagFilterExpression": {
"type": "string"
},
```

Reply:
> **question** ❓ Is **tagfilters** the same as **tagFilterExpression**? The examples show this:
> 
> ```
> "tagFilterExpression": {
>     "type": "EXPRESSION",
>     "logicalOperator": "AND",
>     "elements": []
>   },
> ```
> 
> The model shows this:
> 
> ```
> "tagFilterExpression": {
> "type": "string"
> },
> ```

Yes, actually `tagFilter` and `tagFilterExpression` both inherit from `tagFilterExpressionElement`, though this is more of an implementation detail

Reply:
Should the doc be updated to show **tagFilterExpression** then? Or maybe the examples should be changed to match the doc?

Reply:
I'm actually more inclined towards removing this part. I've tried to push as much documentation on the fields themselves so that documentation is closer to the model. I might still give a few pointers such as where to get tags and metrics.

Reply:
Sounds good.